core/server/data/migrations/versions/4.43/2022-04-08-14-41-add-mrr-to-subscriptions.js

```diff
@@ -0,0 +1,8 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('members_stripe_customers_subscriptions', 'mrr', {
```

Comment:
Are we sure we want to go with MRR vs ARR?

Reply:
Discussed in https://ghost.slack.com/archives/C02G9E68C/p1649667605432439?thread_ts=1649408400.454149&cid=C02G9E68C

tl;dr: we'll use MRR.